packages/interactivity/package.json

```diff
@@ -24,6 +24,7 @@
 	},
 	"main": "build/index.js",
 	"module": "build-module/index.js",
+	"wp-module": "build-module/index.js",
```

Comment:
I was thinking about how we could smoothly transition to ES Modules here. What if we start using [`"type": "module"`](https://nodejs.org/api/packages.html#type) and completely remove the obsolete support for CJS here. This way we could easily distinguish what the package build process should do about packages:

`"type": "module"` + `exports` - ES Module only
`"main": "some/path.js" - CJS
`"module": "some/path.js" - ES Module

More on [package entry points here](https://nodejs.org/api/packages.html#package-entry-points). I'm still processing it all, so this is more of me documenting the efforts.

Reply:
Interesting fact, we use `module` field when deciding whether to build the package:

https://github.com/WordPress/gutenberg/blob/e40447e274390fa76ae01d2db19be2dc7db6504f/bin/packages/get-packages.js#L45

Reply:
We never check whether CJS is required or not, but we only generate the folder in the production mode to speed up `npm run dev` as CJS isn't consumed by the Gutenberg plugin:

https://github.com/WordPress/gutenberg/blob/e40447e274390fa76ae01d2db19be2dc7db6504f/bin/packages/build-worker.js#L32-L37